sacro-app/src/create-window.js

```diff
@@ -73,20 +83,16 @@ const createWindow = async () => {
   });
 
   const result = await dialog.showOpenDialog({
-    title: "Choose ACRO outputs json file",
-    properties: ["openFile"],
+    title: "Choose directory containing outputs you wish to review",
```

Comment:
I think Windows tends to use "folder" rather than directory. However, I appreciate that's super picky and might not be worth changing.

Reply:
Hmm, yeah, perhaps that would be better.  I may change it in later PR